composer.json

```diff
@@ -1,12 +1,17 @@
 {
     "name": "friendsofsymfony/message-bundle",
     "type": "symfony-bundle",
-    "description": "Provides user-to-user messaging features for your Symfony2 application.",
+    "description": "Provides user-to-user messaging features for your Symfony application.",
     "keywords": ["symfony", "message", "messaging"],
     "homepage": "https://github.com/FriendsOfSymfony/FOSMessageBundle",
     "license": "MIT",
     "authors": [
         {
+            "name": "Titouan Galopin",
```

Comment:
You're part of the friendsofsymfony community :) that's not against but we can't add everyone to this file and imo only those that were at the very beginning of the bundle belongs to this section.